### PR TITLE
0.5.4 Main

### PR DESCRIPTION
**Pull request by JamesRobertsonGames**
[py-partiql-parser-feedstock](https://github.com/AnacondaRecipes/py-partiql-parser-feedstock) 0.5.4 

**Destination channel:** defaults

### Links

- [PKG-4577](https://anaconda.atlassian.net/browse/PKG-4577) 
- [Upstream repository](https://github.com/getmoto/py-partiql-parser)
- [Upstream changelog/diff](https://github.com/getmoto/py-partiql-parser/releases/tag/0.5.4)
- Relevant dependency PRs:
  - https://github.com/AnacondaRecipes/moto-feedstock/pull/36

### Explanation of changes:
- new feedstock

[PKG-4577]: https://anaconda.atlassian.net/browse/PKG-4577?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]

**Review by cbouss — COMMENTED** [review]
Can you make the feedstock public and add a feedstock license and readme?
You can copy/paste from https://github.com/AnacondaRecipes/datadog-api-client-feedstock

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - py_partiql_parser
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:31]:
```suggestion
test:
  source_files:
    - tests
  imports:
    - py_partiql_parser
  commands:
    - pip check
    - pytest -v tests
  requires:
    - pip
    - pytest
```
We can run the tests. They are very small and very fast to run (less than 1 second based on GitHub Actions).

**Comment by JamesRobertsonGames** [comment]
@JeanChristopheMorinPerso the tests don't seem to run locally on CI with the config provided, is this because of the recipe or because of how the tarball has been packaged? (the last one is a bit of a guess on my side, usually tests run easily enough in my experience)

**Comment by JeanChristopheMorinPerso** [comment]
The tests are simply not part of the sdist, see https://inspector.pypi.io/project/py-partiql-parser/0.5.4/packages/db/cb/1a8a3ba98ef891c29b861e1f69c1687217a55abd9036a974e3cf17cd7bac/py_partiql_parser-0.5.4.tar.gz/.

You can skop the tests then for this package.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**